enum.h

```diff
@@ -349,7 +351,7 @@ BETTER_ENUMS_CONSTEXPR_ static T* _or_null(optional<T*> maybe)
 
 template <typename T, typename U>
 BETTER_ENUMS_CONSTEXPR_ U
-continue_with(T ignored BETTER_ENUMS_UNUSED, U value) { return value; }
+continue_with(T BETTER_ENUMS_UNUSED_WRAPPER(ignored) BETTER_ENUMS_UNUSED, U value) { return value; }
```

Comment:
I didn't realise there was a failure in the clang build, I have pushed a fix that should hopefully address that.

I hadn't removed the unused variable because I thought there was a reason it was left there to begin with; I thought maybe GCC needed it. Was there a reason the `BETTER_ENUMS_UNUSED` macro was created in the first place instead of the variables simply being omitted?

Anyway, I've also removed the unused variable name in the commit and pushed that as well, I'll see how the build goes. Should I remove the used variable definition in the operator overload below as well?